crates/gdtk-ast/src/ast.rs

```diff
@@ -3,24 +3,29 @@
 
```

Comment:
As a future improvement, the ast definition should be split up into multiple files